modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/table-view/DropZone.es.js

```diff
@@ -100,17 +101,18 @@ const DropZone = ({fields, onAddFieldName, onRemoveFieldName}) => {
 				columns={fields.map(({label, name}) => ({
 					key: label,
 					value: (
-						<div className="container p-0">
-							<div className="align-items-center row">
-								<div className="autofit-col-expand col">
+						<ClayLayout.ContainerFluid className="p-0">
+							<ClayLayout.ContentRow verticalAlign="center">
```

Comment:
I've actually tried this and `align-items-center` is not the same of `autofit-row-center` but maybe in this case we change use it (we should check the UI) https://codepen.io/eduardoallegrini/pen/abvrjdr

Reply:
I think the example `codepen` is not a good example because we don't use fix heights, I've created a fork to illustrate better what we need, we want to equal vertical-align content because of some contents heights can be higher than others:

https://codepen.io/marcoscv/pen/ExPKaMo?editors=1100